.github/scripts/pr_list.mjs

```diff
@@ -0,0 +1,39 @@
+// @ts-check
+import * as url from 'node:url';
+import * as fs from 'node:fs/promises';
+import * as path from 'node:path';
+import * as process from 'node:process';
+import { getCurrentHistorySection, output } from './util.mjs';
+
+const { GITHUB_OUTPUT = '' } = process.env;
+if (GITHUB_OUTPUT.length === 0) throw new Error('Expected GITHUB_OUTPUT file path');
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
+const historyFilePath = path.join(__dirname, '..', '..', 'HISTORY.md');
+
+/**
+ * @param {string} history
+ * @returns {string[]}
+ */
+function parsePRList(history) {
+  const prRegexp = /node-mongodb-native\/issues\/(?<prNum>\d+)\)/giu;
+  const lines = history.split('\n');
+  const prs = [];
+
+  for (const line of lines) {
```

Comment:
you could inline this with something like `prs = lines.map(line => prRegexp.exec(line)?.match?.groups?.prNum).filter(prNum => !!prNum)`

Reply:
Sure, done